Assets/Scripts/Player/InputParser.cs

```diff
@@ -83,6 +87,11 @@ private void Spam(InputAction.CallbackContext context)
             if (currentPaw)
                 currentPaw.MoveForward();
         }
+        private void Step(InputAction.CallbackContext context)
+        {
+            if (cucumberCat)
+                cucumberCat.PerformStep();
+        }
```

Comment:
```suggestion
        
        private void Step(InputAction.CallbackContext context)
        {
            if (cucumberCat)
                cucumberCat.PerformStep();
        }
```
Graag ruimte om funties behouden, [convention code](https://github.com/bas-boop/ProjectIgnite/blob/main/CodeConvention.md#functions-1)